json/json.php

```diff
@@ -344,16 +347,25 @@ function json_last_error_msg () {}
 define('JSON_ERROR_UTF16',10);
 
 /**
- * @since 7.2
- **/
-define('JSON_INVALID_UTF8_IGNORE',1048576);
+ * @since 7.3
+ */
+define('JSON_THROW_ON_ERROR', 4194304);
 
 /**
- * @since 7.2
+ * Class JsonException
+ *
+ * <p>A new flag has been added, JSON_THROW_ON_ERROR, which can be used with
+ * json_decode() or json_encode() and causes these functions to throw a
+ * JsonException upon an error, instead of setting the global error state that
+ * is retrieved with json_last_error(). JSON_PARTIAL_OUTPUT_ON_ERROR takes
+ * precedence over <b>JSON_THROW_ON_ERROR</b>.
+ * </p>
+ *
+ * @since 7.3.0
+ * @link https://wiki.php.net/rfc/json_throw_on_error
  */
-define('JSON_INVALID_UTF8_SUBSTITUTE',2097152);
```

Comment:
Same here.

Reply:
Both constants where already in that json file, so I removed the duplicates